src/blueprints/map/static/css/map.css

```diff
@@ -10,22 +10,22 @@
 
 #character {
     position: absolute;
-    top: 200px; /* Adjust as needed */
-    left: 200px; /* Adjust as needed */
+    top: 200px;
+    left: 200px;
     text-align: center;
 }
 
 #playerIcon {
     width: 50px;
     height: 50px;
     cursor: pointer;
-    display: block; /* Makes the icon a block element */
-    margin: 0 auto; /* Centers the icon horizontally */
+    display: block;
+    margin: 0 auto; 
 }
 
 #healthBarContainer {
     position: absolute;
-    top: -20px; /* Adjust as needed */
+    top: -20px;
```

Comment:
Not really relevant, but if possible try to use `rem` values instead of `px`. rem scales with user specified font size. By default  `1 rem` ==  `16 px`

Using display: flex and trying to keep everything relative often provides responsive layouts for free.

Reply:
Yes, I should use these especially because I'm majoring in Mobile Solutions. :P I've been trying to avoid doing anything UI-related but there is really no excuse for not using em and rem.